index.js

```diff
@@ -41,7 +41,8 @@ function logIn (fields) {
   .then(body => {
     if (body.error) {
       log('debug', `${body.error.code}: ${body.error.libelle}`)
-      throw new Error('LOGIN_FAILED')
+      log('error', 'LOGIN_FAILED')
```

Comment:
Since this connector uses cozy-konnector-libs v3, it would be better to use the new [terminate](https://github.com/cozy/cozy-konnector-libs/blob/master/docs/api.md#baseKonnector+terminate) method of the BaseKonnector : 

```javascript
this.terminate('LOGIN_FAILED')
```

But you have to bind the login function to the connector.

Reply:
Ok thanks, do you have an example of a binded login function somewhere ? I don't see how to bind it easly without a deep refactoring:
```js
module.exports = new BaseKonnector(function fetch (fields) {
  return logIn.(fields) // No konnector context here
  [...]
})
```

Reply:
Actually, the fetch function is already binded by the BaseKonnector itself (https://github.com/cozy/cozy-konnector-libs/blob/master/libs/BaseKonnector.js#L45).

That is why you can do : 
```javascript
return logIn.bind(this)(fields)
```

Reply:
Ok, thanks ! 👍

Reply:
So my
```js
  return logIn.call(this, fields)
```
was OK, only my `cozy-konnector-libs` was not up-to-date.

Reply:
Ok, I see